libs/utils/platforms/juno_energy.py

```diff
@@ -90,7 +90,7 @@ def a57_cpu_node(cpu):
                            active_states=a57_cpu_active_states,
                            idle_states=a57_cpu_idle_states)
 
-juno_energy = EnergyModel(
+juno_r0_energy = EnergyModel(
```

Comment:
Should we not rename the file too? That's for consistency as well as "to make it obvious" as you say in the commit message..

Reply:
Done.